### PR TITLE
Improving the wording of Regulation 11g

### PR DESCRIPTION
**Pull request by mirandapedro**
As pointed out in issue #874 , Regulation 11g currently doesn't make it clear that Delegates have to ensure copies of the Regulations and Guidelines are available to all competitors and officials, and not only to himself.

Rather than adding a Guideline to clear this up, we can fix this by simply adding the Regulations must be available to "officials and compeitors" with the appropriate changes in propositions and wording to make this work.

This is also in harmony with the discussions in [this](https://forum.worldcubeassociation.org/t/removing-11g/1303) topic of the WCA Forum and this issue in the `thewca/wca-documents` repository.

### CODE REVIEW

**Review by Ivan-Brigidano — COMMENTED** [review]
LGTM. But let's wait for other WRC members approval.

**Comment by hanwu85** [comment]
LGTM, too.

**Comment by mirandapedro** [comment]
Could someone give the PR an approving review? 

That way I can merge it into the draft.

**Review by Ivan-Brigidano — APPROVED** [review]

**Comment by mirandapedro** [comment]
Thanks!

**Pull request merged**
